llvm/include/llvm/SYCLLowerIR/CallgraphUtils.h

```diff
@@ -0,0 +1,50 @@
+//===------------ CallgraphUtils.h - Callgraph utility functions
```

Comment:
I am thinking if it might be a good idea to name this with something more generic as SYCLUtils.h. This would enable us to add more varied 'utility' functions in the future. 

Thanks

Reply:
good catched, fixed